ruby/red-parquet/test/test-arrow-table.rb

```diff
@@ -41,12 +41,16 @@ def setup
   def test_save_load_path
     tempfile = Tempfile.open(["red-parquet", ".parquet"])
     @table.save(tempfile.path)
-    assert_equal(@table, Arrow::Table.load(tempfile.path))
+    assert do
+      @table.equal_metadata(Arrow::Table.load(tempfile.path), false)
+    end
```

Comment:
@shiro615 Does it mean that `@table.save` doesn't store metadata?
Can we store metadata by `@table.save`?

Reply:
@kou 
`@table.save` store metadata.
```rb
[3] pry(#<TestArrowTableReader>)> Arrow::Table.load(tempfile.path).schema
=> #<Arrow::Schema:0x7f7f1f986750 ptr=0x7f7f2118d0e0 count: uint8
-- metadata --
PARQUET:field_id: 1
visible: bool
-- metadata --
PARQUET:field_id: 2>
```

`Arrow::Table.new` doesn't store metadata.

```rb
[4] pry(#<TestArrowTableReader>)> @table.schema
=> #<Arrow::Schema:0x7f7f1f97b5a8 ptr=0x7f7f1f063cb0 count: uint8
visible: bool>
```

Reply:
Thanks.
I understand.